src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -48,14 +48,20 @@ <h4 class="panel-title border-bottom-small" translate>
               <span ng-repeat-end></span>
               <label class="btn btn-default-form u-textLeft custom-amount">
                 <div class="form-group form-group-default u-inline">
-                  <input class="form-control form-control-subtle number" type="text" name="amount"
+                  <input class="form-control form-control-subtle number"
```

Comment:
input type number can't be used here.  Jason doesn't want the up/down arrows to show on the input, that's why it was using text.

A validation function may need to be written.